src/core/Geometry.js

```diff
@@ -785,8 +785,10 @@ Object.assign( Geometry.prototype, EventDispatcher.prototype, {
 			return;
 
 		}
-
-		mesh.matrixAutoUpdate && mesh.updateMatrix();
+		
+		if ( mesh.matrixAutoUpdate ) {
```

Comment:
That would work, they don't like bool && function as it's not really a "valid" statement it's more of undefined behavior